packages/@aws-cdk/aws-ec2/lib/security-group.ts

```diff
@@ -399,6 +422,12 @@ export class SecurityGroup extends SecurityGroupBase {
 
     this.allowAllOutbound = props.allowAllOutbound !== false;
 
+    if (props.disableInlineRules !== undefined) {
```

Comment:
maybe this should have `&& props.disableInlineRules !== null`

I'm open to changing, but not sure if its necessary.

Reply:
There is a test case for this, so I think its covered.